bin/options.js

```diff
@@ -69,9 +70,26 @@ function getOptions() {
     return;
   }
 
+  try {
+    const testDir = JSON.parse(
+      fs.readFileSync(path.join(process.cwd(), 'package.json'), 'utf8')
+    ).directories['test'];
+
+    if (testDir && testDir !== 'test') {
+      process._testDirectory = testDir;
+    }
+  } catch (ignore) {
+    // NOTE: should console.error() and throw the error
+  }
+
```

Comment:
So all of this should go into a new exported `getTestDirectory` function. Your implementation above should only be done once, caching the result.


Don't add properties to objects you don't own.

Reply:
ok, done.